Assets/Scripts/BlockBehaviour.cs

```diff
@@ -10,10 +10,15 @@ public class BlockBehaviour : MonoBehaviour
 
     [Range(0f, 1f)]
     public float SkinToLengthRatio = 0.1f;
-    public float Speed = 1f;
+
+    private float CurrentSpeed;

```

Comment:
remember to follow our styling. You can download https://www.jetbrains.com/resharper/. its 👍

Reply:
If you haven't already done so.